maintainers-circle/hosting.md

```diff
@@ -0,0 +1,87 @@
+# Hosting Maintainers Circle
+
+Hosts are responsible for planning and hosting the circle topic they are
+running. Below is guidance to help in this process. 
+
+## Planning
+
+### Curating content
+1. Check on the [README] and [issue backlog] to see what topics are in the queue
+ first. If no topics are left, initiate ideas with the sig in slack. Next step, 
+ ask directly in the #maintainers-circle channel as to what folks would like to 
+ discuss/learn.
+ 2. Add a short bio for the speakers
+
+Content should not be a sales pitch of any kind. It should match with the goals
+and intent of the group and have at least one 15 minute break out session to so
```

Comment:
s/to//

Don't breakout rooms make sense if if the attendees want to break up at least into 2 groups? If the main zoom meeting is interactive enough, is  a breakout room mandatory?

Reply:
The main room only instead of breakout rooms is fine if under 15 folks show up. anything over and many folks will not have the time to engage or speak. most of our sessions have 20-30 folks.